### PR TITLE
Otp verification

### PR DESCRIPTION
**Pull request by vivekwebdesk**
02-01-2019 commit for OTP screen

### CODE REVIEW

File: my-app/screens/Login.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
     TextInput
 } from "react-native";
 
-import Background from "../Background";
+import Background from "../custom/Background";
```
Comment (sumitWebdesk) [inline my-app/screens/Login.js]: Please make it proper folder name

File: my-app/screens/OTP.js [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+  Button,
+  Text,
+  View,
+  StatusBar,
+  ScrollView,
+  TouchableOpacity,
+  Alert
+} from "react-native";
+import CodeInput from 'react-native-confirmation-code-input';
+import Background from "../custom/Background";
```
Comment (sumitWebdesk) [inline my-app/screens/OTP.js]: please make it proper folder name

File: my-app/screens/OTP.js [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+
+export default class OTP extends React.Component {
+
+  mobileNumber={myMobileNumber:'+91 8888888888'}
```
Comment (sumitWebdesk) [inline my-app/screens/OTP.js]: please make it phone number dynamic

File: my-app/screens/Password.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ import {
     Alert,
     TextInput
 } from "react-native";
-import Background from "../Background";
+import Background from "../custom/Background";
```
Comment (sumitWebdesk) [inline my-app/screens/Password.js]: Please make it file name Proper

File: my-app/screens/OTP.js (hunk 3 above)
Comment (vivekwebdesk) [reply my-app/screens/OTP.js]: This is a demo app. I can't add a dynamic number.

Comment (sumitWebdesk) [reply my-app/screens/OTP.js]: okay but please make it proper when API is done

**Pull request closed without merging**